udata_hydra_csvapi/error.py

```diff
@@ -0,0 +1,28 @@
+import json
+import sentry_sdk
+from aiohttp import web
+from udata_hydra_csvapi import config
+
+
+class QueryException(web.HTTPException):
+    """Re-raise an exception from postgrest as aiohttp exception"""
+
+    def __init__(self, status, error_code, title, detail) -> None:
+        self.status_code = status
+        error_body = {
+            "errors": [{"code": error_code, "title": title, "detail": detail}]
+        }
+        super().__init__(content_type="application/json", text=json.dumps(error_body))
+
+
+def handle_exception(status, title, detail, resource_id):
```

Comment:
You can add type hints here :)
Seems like detail could be a string or a dict right?